seatunnel-e2e/seatunnel-spark-connector-v2-e2e/connector-elasticsearch-spark-e2e/src/test/java/org/apache/seatunnel/e2e/spark/v2/elasticsearch/ElasticsearchIT.java

```diff
@@ -0,0 +1,179 @@
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.shaded.org.apache.commons.lang3.ThreadUtils;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.UnknownHostException;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class ElasticsearchIT extends SparkContainer {
+
+    private List<String> testDataset;
+
+    private ElasticsearchContainer container;
+
+    private EsRestClient esRestClient;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ElasticsearchIT.class);
+
+    @BeforeEach
+    public void startMongoContainer() throws Exception {
+        container = new ElasticsearchContainer(DockerImageName.parse("elasticsearch:6.8.23").asCompatibleSubstituteFor("docker.elastic.co/elasticsearch/elasticsearch"))
+                .withNetwork(NETWORK)
+                .withNetworkAliases("elasticsearch")
+                .withLogConsumer(new Slf4jLogConsumer(LOGGER));
+        container.start();
+        LOGGER.info("Elasticsearch container started");
+        esRestClient = EsRestClient.createInstance(Lists.newArrayList(container.getHttpHostAddress()), "", "");
+        testDataset = generateTestDataSet();
+        createIndexDocs();
+
+    }
+
+    /**
+     * create a index,and bulk some documents
+     */
+    private void createIndexDocs() {
+        StringBuilder requestBody = new StringBuilder();
+        Map<String, String> indexInner = new HashMap<>();
+        indexInner.put("_index", "st");
+        indexInner.put("_index", "st");
+
+        Map<String, Map<String, String>> indexParam = new HashMap<>();
+        indexParam.put("index", indexInner);
+        String indexHeader = "{\"index\":{\"_index\":\"st_index\",\"_type\":\"st\"}\n";
+        for (int i = 0; i < testDataset.size(); i++) {
+            String row = testDataset.get(i);
+            requestBody.append(indexHeader);
+            requestBody.append(row);
+            requestBody.append("\n");
+        }
+        esRestClient.bulk(requestBody.toString());
+    }
+
+    @Test
+    public void testElasticsearch() throws IOException, InterruptedException {
+        Container.ExecResult execResult = executeSeaTunnelSparkJob("/elasticsearch/elasticsearch_source_and_sink.conf");
+        Assertions.assertEquals(0, execResult.getExitCode());
+        List<String> sinData = readSinkData();
+        Assertions.assertIterableEquals(
+                testDataset,
+                sinData
+        );
+    }
+
+    private List<String> generateTestDataSet() throws JsonProcessingException, UnknownHostException {
+        String[] fiels = new String[]{
```

Comment:
```suggestion
        String[] fields = new String[]{
```

Reply:
All your suggestions is improved.Thinks.